client/js/components/user/DpUserListExtendedItem.vue

```diff
@@ -279,10 +285,6 @@ export default {
       }
 
       return dpApi.patch(url, {}, payload)
```

Comment:
The `options` you added above need to be sent as a separate parameter, not inside `payload`. So it should be

`return dpApi.patch(url, {}, payload, options)`

and `options` needs to be declared as a separate variable.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/2538/commits/c5093396e81579ffa345dc9c20fdb3b320a3905f

Reply:
https://github.com/demos-europe/demosplan-core/pull/2538/commits/b98581f7ba4601cf92adfe71086f322bf88f3640